ui/service.go

```diff
@@ -206,17 +206,17 @@ type Service interface {
 	// ListThings retrieves things owned/shared by a user.
 	ListThings(token, status string, page, limit uint64) ([]byte, error)
 	// ViewThing retrieves information about the thing with the given ID.
-	ViewThing(token, id string) ([]byte, error)
+	ViewThing(token, thingID string) ([]byte, error)
```

Comment:
```suggestion
	ViewThing(token, id string) ([]byte, error)
```
This is because what we are viewing is definitely a thing